src/OSPSuite.UI/Binders/CurveBinder.cs

```diff
@@ -303,67 +304,73 @@ private void updateMarker(SimpleMarker marker)
 
       private void refreshData()
```

Comment:
I'm not quite seeing the need for this method at all from the investigation I did. It feels like the datacolumns and curves are readonly in the charts. after initialization. I don't think a `CurveBinder` is ever updated with new data in the underlying datarepository.

Reducing the calls to this did not seem to affect the actual repaint time of the chart, but for curves of 10k it takes time to repopulate the datatable, and 10 curves of 10k when you really only changed 1 can be significant waste.

With this update this refresh is not called when editing curve properties, but it will still be called for other chart updates. But I don't think it's really needed.

Reply:
Hmm, better check what happens when you change the axis units for example. Might need to recalculate the data points in that case.

Reply:
Yeah, you need it when there are scaling changes on the axes. Still, when we change curve properties we can skip this refresh.